indy_common/types.py

```diff
@@ -142,6 +143,53 @@ class GetContextField(MessageValidator):
     )
 
 
+class RsSchemaMetaField(MessageValidator):
+    schema = (
+        (RS_META_TYPE, ConstantField(RS_SCHEMA_META_TYPE)),
+        (RS_META_NAME, LimitedLengthStringField(max_length=NAME_FIELD_LIMIT)),
+        (RS_META_VERSION, VersionField(version_cls=RsMetaVersion)),
+    )
+
+
+class RsSchemaField(MessageValidator):
+    SCHEMA_IS_STRICT = False
+    schema = (
+        (RS_JSON_LD_TYPE, LimitedLengthStringField(max_length=NAME_FIELD_LIMIT)),
+        (RS_JSON_LD_ID, LimitedLengthStringField(max_length=NAME_FIELD_LIMIT)),
+    )
+    _base_types = None
+
+    def __init__(self, max_size=None, **kwargs):
+        if max_size is not None:
+            if not isinstance(max_size, int):
+                raise PlenumTypeError('max_size', max_size, int)
```

Comment:
Are there tests for this exception raises?

Reply:
There are not any tests for this error. I will add one.